classes/Helper.php

```diff
@@ -0,0 +1,122 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Version information
+ *
+ * @package    report_rolessitemap
+ * @copyright  2022 Andreas Schenkel
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+namespace report_rolessitemap;
+use moodle_url;
+use report_rolessitemap\Misc;
+
+defined('MOODLE_INTERNAL') || die();
+
+class Helper {
+    /**
+     * Shows all categories and the assigned users with their roles
+     *
+     * @return void
+     * @throws \dml_exception
+     */
+    public function rendercategoricesandroles(): void {
+        GLOBAL $DB, $OUTPUT;
```

Comment:
"GLOBAL" should be lowercase -> "global"

Reply:
fixed